docs/content/docs/updates/v1.0.0.md

```diff
@@ -6,12 +6,39 @@ summary: |
   The 1.0 release is the first major release of the Fluid Framework.
 ---
 
-## Patch Updates
+# 1.3.0
 
-### Update 1.0.1:
+## @fluidframework/fluid-static
+
+We are now surfacing errors on FluidContainer “disposed” event, so the application can appropriately react to faulty conditions, if container was closed due to an error.
+
+## Notable bug fixes
+
+- If an AzureClient is used to open multiple documents/containers, the internal in-memory cache was not handling that scenario correctly, so the "latest" snapshot would be polluted by the first document loaded. This has been fixed.
+
+- Fix url deps and add eslint rules to not allow core nodejs modules on lts.
```

Comment:
@tylerbutler @RishhiB could you clarify this change/fix?

Reply:
```suggestion
- Fluid packages no longer require users of WebPack 5 and other modern bundlers to polyfill the node.js `assert`, `url` and `events` APIs. Note that the `buffer` API must still be polyfilled. This is tracked by [issue 8725](https://github.com/microsoft/FluidFramework/issues/8725).
```